src/runtime/providers/ipx.ts

```diff
@@ -12,7 +12,7 @@ const operationsGenerator = createOperationsGenerator({
     quality: 'q',
     background: 'b'
   },
-  joinWith: ',',
+  joinWith: '_',
```

Comment:
I think this breaks placeholder generation for IPX as it generates the following URL when using the `placeholder` attribute:

```
/_ipx/q_50,s_10x10/images/slightly_smiling_face_3d.png
```

which is a valid IPX URL.

Reply:
That's possible. If there's a better solution feel free to close this out.

**Note:** In addition to fixing the described bug, this PR also made it so that the downloaded directory names don't have commas or ampersands in them. Not sure if there's a better way to accomplish this, but to me it feels a bit hacky to have these characters in folder names. Underscores or hyphens seem cleaner. Also not sure if there are any SEO implications to consider here.